openreview/conference/templates/paperRankingWebfield.js

```diff
@@ -57,25 +57,28 @@ function load() {
       });
   });
 
-  var tagInvitationsP = Webfield.getAll('/invitations', {id: PAPER_RANKING_ID, type: 'tags'}).then(function(invitations) {
+  var paperRankingInvitationP = Webfield.getAll('/invitations', {id: PAPER_RANKING_ID, type: 'tags'}).then(function(invitations) {
     return invitations.filter(function(invitation) {
       return invitation.invitees.length;
-    });
+    })[0];
```

Comment:
I think you should check the length of the array first, this could throw an error.

Reply:
I think this webfield is going to be removed.